_data/guides.yml

```diff
@@ -12,6 +12,8 @@ toc:
     path: /docs/getting-started-guides/kubeadm/
   - title: Hello World on Google Container Engine
     path: /docs/hellonode/
+  - title: Downloading kubectl
```

Comment:
We are installing.  I was trying to hit the common SEO terms, but I think linking is probably going to be a bigger win.